pkg/utils/cel/default_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cel
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
This package is currently not expected to be imported

Reply:
I will revert the import and write the tests without using the `testify` package.